Makefile

```diff
@@ -52,13 +62,17 @@ extra-clean: clean
 	go clean -i $(PKG_GOX)
 	go clean -i $(PKG_GOLANGCI_LINT)
 
+# fmt will fix the golang source style in place, and returns an error code if any source code are changed.
```

Comment:
I think it only returns an error, if it couldn'f fix a formatting error or if it somehow failed halfway through

Reply:
Good catch.  The 2nd half of the comment is meant for check-fmt.  Will fix it.